project/Paradox.scala

```diff
@@ -48,8 +48,6 @@ object Paradox {
         "index.html",
         // Page that recommends Alpakka:
         "camel.html",
-        // Page linked to from many others, but not in a TOC
-        "typed/failure-detector.html",
```

Comment:
this can be good to know; pages must be in some `@@@ index` otherwise the ToC will be strange when navigating to that page